generators/Program.cs

```diff
@@ -20,14 +21,29 @@ private static void SetupLogger()
                 .CreateLogger();
         }
 
-        private static void Generate(Options options)
+        private static void RegenerateTestClasses(Options options)
         {
-            Log.Information("Generating tests...");
+            Log.Information("Re-generating test classes...");
+
+            var canonicalDataParser = CreateCanonicalDataParser(options);
```

Comment:
CreateCanonicalDataParser eventually leverages Git2Sharp which can throw an exception if no network connection is found. We definitely cannot continue if the data cannot be retrieved. It might be nice to catch this instead of leaving it unhandled.